build/rust/Cargo.lock

```diff
@@ -2313,9 +2352,9 @@ dependencies = [
 
 [[package]]
 name = "serde_ipld_dagcbor"
-version = "0.2.0"
+version = "0.2.2"
```

Comment:
this bump also doesn't look related so it should be rolled back

Reply:
This is auto-bumped since fvm_shared is updated to 3.3.0
https://crates.io/crates/fvm_shared/3.3.0/dependencies
https://crates.io/crates/fvm_ipld_encoding/0.3.3/dependencies

Reply:
Yes, and we should undo that as we have done in other PRs that made changes to Cargo.lock. cargo updates everything to the latest compatible version by default, but we don't want that. We want to update only what is absolutely necessary so you need to leave out changes that we don't actually need and only include the ones that we do need. After cargo runs you need to revert the changes we don't actually need.

Reply:
Wouldn't cargo overwrite them on every launch?

Reply:
Yes, manual edits of Cargo.lock will have to be redone every time. fvm_ipld_encoding [requires](https://github.com/filecoin-project/ref-fvm/blob/fvm_ipld_encoding%40v0.3.3/ipld/encoding/Cargo.toml#L12) serde_ipld_dagcbor v0.2.2 so cargo will try to reset the back reference.

@bridiver can you confirm that's what you're asking for?

Reply:
@kdenhartog how concerned are you about [this issue](https://github.com/ipld/serde_ipld_dagcbor/pull/2)? The functional difference between our 0.2.0 and the 0.2.2 version fvm specifies is that our current version doesn't enforce the identity [multibase prefix](https://github.com/multiformats/multibase#multibase-table) for the cid, allowing base64, etc. encodings to be interpreted as binary.

Reply:
@bridiver said on slack that bumps like this were ok, so afaict we've agreed this change is acceptable.

Reply:
I'd prefer bumping this personally. This seems functionally useful for us.